--- /home/oliver/mounts/evdev	2020-09-26 17:22:19.337836937 +0100
+++ evdev	2020-09-04 18:20:25.000000000 +0100
@@ -1049,6 +1049,7 @@
   ctrl:menu_rctrl	=	+ctrl(menu_rctrl)
   ctrl:ralt_rctrl	=	+ctrl(ralt_rctrl)
   ctrl:swap_lalt_lctl	=	+ctrl(swap_lalt_lctl)
+  ctrl:swap_ralt_rctl = +ctrl(swap_ralt_rctl)
   ctrl:swap_lwin_lctl	=	+ctrl(swap_lwin_lctl)
   ctrl:swap_rwin_rctl	=	+ctrl(swap_rwin_rctl)
   ctrl:swap_lalt_lctl_lwin  =	+ctrl(swap_lalt_lctl_lwin)
