--- /home/oliver/mounts/pc	2020-09-26 17:21:32.997901614 +0100
+++ pc	2020-09-26 16:26:37.000000000 +0100
@@ -19,7 +19,8 @@
     key  <TAB> {	[ Tab,	ISO_Left_Tab	]	};
     key <RTRN> {	[ Return		]	};
 
-    key <CAPS> {	[ Caps_Lock		]	};
+    // key <CAPS> {	[ Caps_Lock		]	};
+    key <CAPS> {    [ Hyper_L       ]   };         //add Hyper_L mapping
     key <NMLK> {	[ Num_Lock 		]	};
 
     key <LFSH> {	[ Shift_L		]	};
@@ -33,9 +34,10 @@
 
     // Beginning of modifier mappings.
     modifier_map Shift  { Shift_L, Shift_R };
-    modifier_map Lock   { Caps_Lock };
+    // modifier_map Lock   { Caps_Lock };
     modifier_map Control{ Control_L, Control_R };
     modifier_map Mod2   { Num_Lock };
+    modifier_map Mod3   { Hyper_L, Hyper_R };      //add modifier for Hyper (Mod3)
     modifier_map Mod4   { Super_L, Super_R };
 
     // Fake keys for virtual<->real modifiers mapping:
@@ -53,7 +55,7 @@
     modifier_map Mod4   { <SUPR> };
 
     key <HYPR> {	[ NoSymbol, Hyper_L	]	};
-    modifier_map Mod4   { <HYPR> };
+    modifier_map Mod3   { <HYPR> };                  //set Hyper modifier to Mod3
     // End of modifier mappings.
 
     key <OUTP> { [ XF86Display ] };
